conf/inventory/ibm-eu-rhel-10.0.yaml

```diff
@@ -0,0 +1,111 @@
+---
+version_id: 10.0
+id: rhel
+instance:
+  create:
+    image-name: rhel-10-server-amd64-kvm
```

Comment:
shouldn't this be `rhel-100-server-amd64-kvm`?

Reply:
fixed